### PR TITLE
HCID encoded keys

### PR DESCRIPTION
**Pull request by ddd-mtl**
keybundle stores pub keys in HCID format.
Removed pub key encoding in gossip since not necessary anymore.

Match naming in rust dpki:
bundle -> Blob
keypair -> KeyBundle

### CODE REVIEW

File: lib/hc-dpki/keybundle.test.js [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    try {
+      p.sign(Buffer.from('hello'))
+    } catch (e) {
+      await p.destroy()
+      return
+    }
+    await p.destroy()
+    throw new Error('expected exception, got success')
+  })
+})
```
Comment (ddd-mtl) [inline lib/hc-dpki/keybundle.test.js]: in this file I only did the same renames and changed the harcoded keys

File: docs/hc-dpki/seed.md [hunk 2]
```diff
@@ -147,21 +147,21 @@ This root seed should be pure entropy
...
-| index | <code>number</code> | 
+| index | <code>number</code> |
 
 <a name="RootSeed.newRandom"></a>
 
 ### RootSeed.newRandom()
 Get a new, completely random root seed
 
-**Kind**: static method of [<code>RootSeed</code>](#RootSeed)  
+**Kind**: static method of [<code>RootSeed</code>](#RootSeed)
```
Comment (ddd-mtl) [inline docs/hc-dpki/seed.md]: looks like my IDE removed all the trailing whitespaces

Comment (neonphog) [reply docs/hc-dpki/seed.md]: did you edit the docs directly? I don't think I hooked up the doc generation again after the lerna removal...

Comment (ddd-mtl) [reply docs/hc-dpki/seed.md]: yes, I did it manually as I forgot there was supposed to be automatic doc generation ^^''

**Pull request merged**